dynet/nodes-affinetransform.cc

```diff
@@ -75,6 +75,11 @@ std::vector<int> AffineTransform::autobatch_concat(const ComputationGraph & cg)
 // much faster than using Eigen's tensor contractions (as of the writing)
 template<class MyDevice>
 void AffineTransform::forward_dev_impl(const MyDevice & dev, const vector<const Tensor*>& xs, Tensor& fx) const {
+#if HAVE_CUDA
```

Comment:
We shouldn't add a check to individual nodes, this will cause code bloat. Is the check added to `forward()` in `nodes-macros.h` not enough?